core/src/lib/res.cc

```diff
@@ -1503,39 +1503,7 @@ void IndentConfigItem(PoolMem& cfg_str,
 
 std::string PrintNumberSiPrefixFormat(ResourceItem* item, uint64_t value_in)
 {
-  PoolMem temp;
```

Comment:
this function will probably yield an unused parameter warning on compile.
You could just use `ResourceItem*` (without a parameter name) to identify the value as unused.
Probably it is even better to remove this function completely and replace with `SizeAsSiPrefixFormat()` where applicable.